include/mxnet/lib_api.h

```diff
@@ -763,9 +818,9 @@ typedef int (*opCallInferType_t)(inferType_t, const char* const*, const char* co
 
 #define MXLIB_OPCALLFCOMP_STR "_opCallFCompute"
 typedef int (*opCallFComp_t)(fcomp_t, const char* const*, const char* const*, int,
-                             const int64_t**, int*, void**, int*, size_t*, int,
-                             const int64_t**, int*, void**, int*, size_t*, int,
-                             xpu_malloc_t, void*);
+                             const int64_t**, int*, void**, int*, size_t*, char**, int*, int,
+                             const int64_t**, int*, void**, int*, size_t*, char**, int*, int,
+                             xpu_malloc_t, void*, void*);
```

Comment:
lets refactor and use the same function signature as the actual function later on